gammapy/estimators/tests/test_utils.py

```diff
@@ -199,6 +233,52 @@ def test_compute_lightcurve_doublingtime():
     )
 
 
+def test_compute_dcf():
+    lighcurve = lc()
+    lightcurve2 = lc2()
```

Comment:
I do not know if that'd make a difference in practice, but shouldn't lc and lc2 be fixtures?